backend/src/main/java/com/redhat/cloud/notifications/models/EmailSubscriptionProperties.java

```diff
@@ -1,8 +1,65 @@
 package com.redhat.cloud.notifications.models;
 
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "email_properties")
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
 public class EmailSubscriptionProperties extends EndpointProperties {
 
+    @NotNull
```

Comment:
A primitive attribute can't be `null`.

Reply:
Apparently the openapi builder doesn't know about this. If i omit this, the result is an optional type.